ui/src/components/admin/fundraise/AdminFundraiseContainer.js

```diff
@@ -0,0 +1,326 @@
+                <FormControl>
+                  <FormLabel>Milestone Name</FormLabel>
+                  <Input
+                    type="text"
+                    value={milestoneName}
+                    onChange={(e) => setMilestoneName(e.target.value)}
+                  />
+                </FormControl>
+                <FormControl>
+                  <FormLabel>Milestone Amount</FormLabel>
+                  <NumberInput>
+                    <NumberInputField
+                      value={milestoneAmount}
+                      onChange={(e) => setAmount(e.target.value)}
+                    />
+                  </NumberInput>
+                </FormControl>
+                <FormControl>
+                  <FormLabel>Milestone Deadline</FormLabel>
+                  <Input
+                    type="datetime-local"
+                    value={milestoneDeadline}
+                    onChange={(e) => setMilestoneDeadline(e.target.value)}
+                  />
+                </FormControl>
+                <Button colorScheme="blue" onClick={handleAddMilestones}>
+                  Add Milestone
+                </Button>
+              </VStack>
+              <UnorderedList>
+                {milestones.map((milestone, index) => (
+                  <Box key={index} w={['100%', '100%', '50%']} my="2">
+                    <Flex>
+                      <Box>
+                        <strong>Milestone Name:</strong>{' '}
+                        {milestone.milestoneName}
+                        <br />
+                        <strong>Milestone Amount:</strong>{' '}
+                        {milestone.milestoneAmount}
+                        <br />
+                        <strong>Milestone Deadline:</strong>{' '}
+                        {milestone.milestoneDeadline}
+                        <br />
+                      </Box>
+                      <Spacer />
+                      <Button
+                        colorScheme="red"
+                        size="sm"
+                        onClick={() => handleRemoveMilestones(index)}
+                        ml={2}
+                        my="auto"
+                        mx="1"
+                      >
+                        Remove
+                      </Button>
+                    </Flex>
+                  </Box>
+                ))}
+              </UnorderedList>
+            </Box>
+            <FormControl w="auto">
+              <Checkbox
+                isChecked={deleted}
+                onChange={(e) => setDeleted(e.target.checked)}
+              >
+                Delete?
+              </Checkbox>
+            </FormControl>
+            <FormControl mt={5}>
+              <Button type="submit" w="full" isLoading={isLoading}>
+                {id && id.length > 0 ? 'UPDATE' : 'SAVE'}
+              </Button>
+            </FormControl>
+            <Button colorScheme="red" w="full" mt={5} onClick={resetHandler}>
+              RESET
```

Comment:
the whole thing crash when i click this :")

Reply:
same as above